recipes/dictlib/meta.yaml

```diff
@@ -1,38 +0,0 @@
-{% set name = "dictlib" %}
-{% set version = "1.1.3" %}
-
-package:
-  name: {{ name|lower }}
-  version: {{ version }}
-
-source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 39ac3c5d02088a7978d330328450e616f207cf3afe24e055bff4a4ad67c3f89d
-
-build:
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install .  -vv
-
-requirements:
-  host:
-    - pip
-    - python
-  run:
-    - python
-
-test:
-  imports:
-    - dictlib
-
-about:
-  home: https://github.com/srevenant/dictlib
-  license: GNU Affero General Public v3
-  license_family: AGPL
-  license_file: LICENSE.txt
-  summary: Dictionary Library including good deep merge and dictionary as objects
-  dev_url: https://github.com/srevenant/dictlib
-
-extra:
-  recipe-maintainers:
-    - Tata17
```

Comment:
Could you please revert those deleted files? Just to avoid any kind of problem.

Reply:
@marcelotrevisani I might have to rebase and creat the PR from scratch again. Thank you!